recipes/webcolors/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.5" %}
+
+package:
+  name: webcolors
+  version: {{ version }}
+
+source:
+  fn: webcolors-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/w/webcolors/webcolors-{{ version }}.tar.gz
+  md5: 2e9da1bbb7a087c6108e05be64640afb
+
+build:
+  number: 0
+  script:
+    - source activate _build  # [unix]
+    - activate _build         # [win]
```

Comment:
I think I remember fixing my activate script as the currently released version of conda/conda-env does not have the activate.d step as part of the `endlocal` line (line = `endlocal && (....)` and therefore the nothing which happens in the activate.d scripts is surviving until the activate script returns.

https://github.com/conda/conda/commit/f531272c24b8c123cab9ddd7c66154cba277d5cc

-> 
bad
```
endlocal & (
    REM Used for deactivate, to make sure we restore original state after deactivation
    SET "CONDA_PATH_BACKUP=%CONDA_PATH_BACKUP%"
    SET "CONDA_OLD_PS1=%CONDA_OLD_PS1%"
    set "PROMPT=%PROMPT%"
    set "PATH=%PATH%"
    set "CONDA_DEFAULT_ENV=%CONDA_NEW_ENV%"
    )
```

workaround:
```
endlocal & (
    REM Used for deactivate, to make sure we restore original state after deactivation
    SET "CONDA_PATH_BACKUP=%CONDA_PATH_BACKUP%"
    SET "CONDA_OLD_PS1=%CONDA_OLD_PS1%"
    set "PROMPT=%PROMPT%"
    set "PATH=%PATH%"
    set "CONDA_DEFAULT_ENV=%CONDA_NEW_ENV%"
    )

REM Run any activate scripts outside of "setlocal"
IF EXIST "%CONDA_DEFAULT_ENV%\etc\conda\activate.d" (
    PUSHD "%CONDA_DEFAULT_ENV%\etc\conda\activate.d"
    FOR %%g in (*.bat) DO @CALL "%%g"
    POPD
)

```